bgcval2/batch_timeseries.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+#
+# Copyright 2015, Plymouth Marine Laboratory
+#
+# This file is part of the bgc-val library.
+#
+# bgc-val is free software: you can redistribute it and/or modify it
+# under the terms of the Revised Berkeley Software Distribution (BSD) 3-clause license.
+
+# bgc-val is distributed in the hope that it will be useful, but
+# without any warranty; without even the implied warranty of merchantability
+# or fitness for a particular purpose. See the revised BSD license for more details.
+# You should have received a copy of the revised BSD license along with bgc-val.
+# If not, see <http://opensource.org/licenses/BSD-3-Clause>.
+#
+# Address:
+# Plymouth Marine Laboratory
+# Prospect Place, The Hoe
+# Plymouth, PL1 3DH, UK
+#
+# Email:
+# ledm@pml.ac.uk
+#
+"""
+.. module:: batch_timeseries
+   :platform: Unix
+   :synopsis: A script to submit slurm scripts time series.
+
+.. moduleauthor:: Lee de Mora <ledm@pml.ac.uk>
+
+"""
+import argparse
+import subprocess
+import os
+import sys
+
+from getpass import getuser
+
+from bgcval2.analysis_compare import load_comparison_yml
+
+
+def get_args():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    parser.add_argument('-y',
+                        '--compare_yml',
+                        nargs='+',
+                        type=str,
+                        help='One or more Comparison Analysis configuration file, for examples see bgcval2 input_yml directory.',
+                        required=True,
+                        )
+
+    parser.add_argument('-c',
+                        '--config-file',
+                        default=os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                             'default-bgcval2-config.yml'),
+                        help='User configuration file (for paths).',
+                        required=False)
+
+    parser.add_argument('--dry_run',
+                        '-d',
+                        default=False,
+                        help='When True: Do not submit the jobs to lotus.',
+                        action=argparse.BooleanOptionalAction,
+                        required=False)
+
+    args = parser.parse_args()
+    return args
+
+
+def submits_lotus(compare_yml, config_user, dry_run=False):
```

Comment:
We're not actually using config_user here, but I'm keeping it for completeness, just in case.